src/course/en/components.json

```diff
@@ -212,7 +212,7 @@
         "_allowFullScreen": true,
         "title": "Media",
         "displayTitle": "Media",
-        "body": "Sometimes we need to make use of rich media to really drive home a point, bring a complex subject to life or just let audio to literally do the talking. This is when the media component comes to the fore.<br><br>Component can be either single or spanned.",
+        "body": "Sometimes we need to make use of rich media to really drive home a point, bring a complex subject to life or simple use audio to - literally - do the talking. This is when the <b>Media</b> component comes to the fore.<br><br>Component can be either single or spanned.",

```

Comment:
If using 'proper' parenthetical dashes rather than hyphens, for consistency with Adapt's use of British English, I'd suggest the en dash (`&ndash;`), The em dash is typically used in American English, and also wouldn't use a space on either side of the dash.

Reply:
Happy to drop the dashes... it was a schoolboyish aversion to the split infinitive that made me add them. But of course there's really nothing wrong with splitting infinitives _let the grammar wars begin_